src/media.rs

```diff
@@ -0,0 +1,41 @@
+use std::{
+    ffi::OsStr,
+    path::{Path, PathBuf},
+};
+
+const SUPPORTED_FORMATS: &[&str] = &["mp4", "gif", "webm", "ogg"];
```

Comment:
```suggestion
const SUPPORTED_FORMATS: &[&str] = &["mp4", "gif", "webm", "ogg", "avi", "mkv", "mpeg"];
```

Although browsers don't really support it, it could be transcoded lately after #2